### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AN-SWAPNIL**

### CODE REVIEW

**Comment by ANSwapnilPro** [comment]
This is a good idea

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
-# L1-T2-Project-Part-2--JavaFX-
```
Comment (ANSwapnilPro) [inline README.md:1]: yeah delete it

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
- This is a movie database project with networking feature. One can log in as a Production Company with a distinct password. See their movie with the trailer and poster. Edit, Add, Delete the company's movie. Transfer to another company in real-time. Change password and logo, etc. Maximum Warning messages are shown here.
```
Comment (ANSwapnilPro) [inline README.md:2]: yeah delete it

**Review by ANSwapnilPro — CHANGES_REQUESTED** [review]
good luck bro

File: README.md (hunk 1 above)
Comment (AN-SWAPNIL) [reply README.md:1]: ok

File: README.md (hunk 2 above)
Comment (AN-SWAPNIL) [reply README.md:2]: ok

**Review by ANSwapnilPro — APPROVED** [review]

**Comment by ANSwapnilPro** [comment]
...

**Pull request closed without merging**